### PR TITLE
set e so we dont ahve to or on exit. log when the lockfile is there. …

### PR DESCRIPTION
**Pull request by kevinkreiser**
…dont need to sed off the quotes jq has a switch for that. have shebang everywhere. clear all logs periodiacally

### CODE REVIEW

File: scripts/backup_tiles.sh [hunk 1]
```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-date_time=`date +%Y_%m_%d_%H%M%S`
-mv $1 $1.$date_time
```
Comment (gknisely) [inline scripts/backup_tiles.sh]: where are we making backups now?

Comment (kevinkreiser) [inline scripts/backup_tiles.sh]: yep i just moved those two  lines into the main script. less scripts makes it easier to figure out what the heck is going on imho

Comment (gknisely) [inline scripts/backup_tiles.sh]: k

**Comment by gknisely** [comment]
:ship:

**Pull request merged**